src/Redux/meal.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const initialState = {
+  loading: false,
+  error: null,
+  meals: [],
+};
+
+export const fetchMeals = createAsyncThunk('meals/fetchMeals', async () => {
+  const response = await axios('https://www.themealdb.com/api/json/v1/1/categories.php');
+  return response.data.categories;
+});
```

Comment:
- [ ] Kindly use a API that returns numerical data, as this is a core requirement. Check the requirements below
![image](https://github.com/AbuTalha3/my-react-app/assets/81932478/cef76921-7196-46b6-ad30-e07eea7b5009)

Here is the link to the requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)

- [ ] Please note that I asked for advice on how to proceed with a senior reviewer and the consensus is that you have to change your API to one that returns some numeric data. If you notice, the API you chose returns text and here we need to display numbers.

Your API
![image](https://github.com/AbuTalha3/my-react-app/assets/81932478/1584a8cc-1ad1-4cc2-b5b8-fbfb9e2a8fbf)

Expected Kind  Of Result
![image](https://github.com/AbuTalha3/my-react-app/assets/81932478/e291bd2e-710e-497b-8789-24861ff6c1e9)


#### Note that the given design has some numbers being retrieved from the API while yours has text. Kindly choose a different API that will display numbers. Now note that because of this your design will change especially on the details page. So please make sure that your design matches the given design.